site/blocks/ExpandableParagraph.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react"
+import classnames from "classnames"
+import ReactDOM from "react-dom"
+
+export const ExpandableParagraph = (
+    props:
+        | {
+              children: React.ReactNode
+              dangerouslySetInnerHTML?: undefined
```

Comment:
I don't think you need to add that to the props type since every React component always has it available, right? Can you try dropping the `dangerouslySetInnerHTML` from the type definition?

Reply:
Nah it's only on `React.IntrinsicElements`. 🙂 

The _actual_ redundant thing here is that we already have an eslint warning to stop people using `children` and `dangerouslySetInnerHTML` at the same time, but I wanted to be explicit just in case.